.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+- package-ecosystem: "docker"
+  directory: "/"
+  schedule:
+    interval: weekly
+  labels:
+    - "area/dependency"
+    - "release-note-none"
+    - "ok-to-test"
+    - "kind/cleanup"
+- package-ecosystem: "docker"
+  directory: "/cmd/webhook"
+  schedule:
+    interval: weekly
+  labels:
+    - "area/dependency"
+    - "release-note-none"
+    - "ok-to-test"
+    - "kind/cleanup"
+- package-ecosystem: gomod
+  directory: "/"
+  allow:
+  - dependency-type: "all"
+  schedule:
+    interval: weekly
+  groups:
+    golang-dependencies:
+      patterns:
+        - "github.com/golang*"
+    k8s-dependencies:
+      patterns:
+        - "k8s.io*"
+        - "sigs.k8s.io*"
+        - "github.com/kubernetes-csi*"
```

Comment:
You can remove "github.com/kubernetes-csi*" since filestore go.mod does not have dependency on that path.

Reply:
Oh ignore this one, it is actually there a few lines above.